ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -11,6 +11,12 @@
     height: calc(100vh - var(--pf-c-page__header--MinHeight) - var(--pf-c-page__header--MinHeight) - var(--pf-c-page__header--MinHeight));
 }
 
+/* Create a stacking context that is higher than the context for the network-graph-toolbar component
+   that is rendered below this toolbar, so that the hierarchy dropdowns will render above everything when open */
+.network-graph-selector-bar {
+    z-index: 350;
```

Comment:
Too bad that it needs z-index instead of simple static display.

Reply:
I did not test other ways of creating a stacking context, in part because z-index is the most explicit way to force it (you can tell what it does even if you're debugging in dev tools, instead of reading the comment in the source code), and in part because of PTSD from the old Internet-Explorer "hasLayout" hacks. 😉